src/Symfony/Component/AstGenerator/Util/AstHelper.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AstGenerator\Util;
+
+use PhpParser\Node\Expr;
+use PhpParser\Node\Scalar;
+
+final class AstHelper
+{
+    /**
+     * Transforms a php value into an AST node.
+     *
+     * @param null|bool|int|float|string|array $value
+     *
+     * @return Expr
+     */
+    public static function value($value)
+    {
```

Comment:
I'm not sure it would support well every edge cases such as non-associative array (`var_export` always prints the keys).
BTW [this part of the code is extracted from nikic library](https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/BuilderAbstract.php#L64-L103).